typescript/sdk/src/providers/SmartProvider/SmartProvider.ts

```diff
@@ -97,6 +98,41 @@ export class HyperlaneSmartProvider
     this.supportedMethods = [...supportedMethods.values()];
   }
 
+  async getPriorityFee() {
+    try {
+      return BigNumber.from(await this.perform('maxPriorityFeePerGas', {}));
+    } catch (error) {
+      return BigNumber.from('1500000000');
```

Comment:
The catch will prevent the retry logic. But if this is the logic you want anyway, add logging and move the number to a const

Reply:
Doesn't this perform at the SmartProvider level? so the retries happen inside `perform`?